theforeman.org/pipelines/test/foreman-installer.groovy

```diff
@@ -23,31 +23,31 @@ pipeline {
                     exclude {
                         axis {
                             name 'ruby'
-                            notValues '2.4'
+                            values '2.5', '2.7'
```

Comment:
Looking at this, shouldn't it exclude `notValues '2.4'` and `values '5.0'`.

Reply:
Alright. Changed it to that. I will screw this up again in the future :)